### PR TITLE
add pagination to admin/users/index using pagy gem

### PR DESCRIPTION
**Pull request by phallstrom**
test plan:
  - bundle install
  - have more than 25 users or adjust the pagination limit
  - view /admin/users
  - see nifty pagination at bottom

### CODE REVIEW

File: app/views/admin/users/index.html.slim [hunk 1]
```diff
@@ -22,4 +22,6 @@ section.section
               td= user.created_at.to_date.to_s(:db)
               td= user.email
               td= user.tz
-              td= link_to "View", admin_user_path(user.id), class: "button"
\ No newline at end of file
+              td= link_to "View", admin_user_path(user.id), class: "button"
```
Comment (brandondees) [inline app/views/admin/users/index.html.slim]: if you've already got a user instance, no need to call `id` on it, rails helpers do it internally with a bit of extra intelligence anyway

**Comment by brandondees** [comment]
screenshots would help

**Comment by phallstrom** [comment]
pagy

**Comment by brandondees** [comment]
seems legit

**Pull request merged**